checks/all modes/timing/CheckUnusedLines.cs

```diff
@@ -72,7 +74,13 @@ public override CheckMetadata GetMetadata() => new BeatmapCheckMetadata()
                         "timestamp - ")
                     .WithCause(
                         "An uninherited line is not placed on a multiple of 4 downbeats away from the previous uninherited line, " +
-                        "and only changes settings which an inherited line could do instead.") }
+                        "and only changes settings which an inherited line could do instead.") },
+
+                { "Problem Useless Inherited",
+                    new IssueTemplate(Issue.Level.Problem,
+                        "{0} Changes nothing.",
```

Comment:
```suggestion
                        "{0} Inherited line changes nothing.",
```
Otherwise ambiguous for timestamps with both uninherited and inherited lines present.

Should also prepend "Uninherited line" for the other templates now that the issue message is simply "timing line" and not "uninherited line".